packages/react-storage/src/styles/storage-browser.css

```diff
@@ -217,7 +217,7 @@
 }
 
 .storage-browser__loading-indicator-icon,
-.storage-browser__table-text-data-cell-icon--action-progress {
+.amplify-storage-browser__table-text-data-cell-icon--action-progress {
```

Comment:
Info: aren't the `.amplify-storage-browser__` classes handled in `ui/src/theme/css/component/StorageBrowser/storageBrowser.scss`?